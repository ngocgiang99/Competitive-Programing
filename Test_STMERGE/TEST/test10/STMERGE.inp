5
10 8
511735941 504468498 760944616 790014554 49438265 678848933 374627618 533390575 
206262419 567310520 687286588 267033478 787896598 266886752 427892187 581037397 
250256777 456092147 709586147 557429201 477503472 65357061 634515578 735472302 
966445712 606971244 252694277 547360227 842940398 168738620 356616488 121764772 
284863184 81709259 449576392 738006629 734681145 105309356 119818764 786513117 
748291440 24842167 953034071 871290367 680162296 849193726 709772111 607914753 
664985496 490927576 909860312 689809240 156929620 411980762 539904792 839058105 
615768178 633353431 663121268 548330265 846133877 708178662 995509648 319588525 
128950051 54006364 109623378 789490665 824996738 821915597 530513118 767593860 
894154454 60086798 97004699 956699249 250844836 705576501 77075244 997732838 
5 2
956415606 482816857 
418301186 351498183 
180598943 542155901 
626452956 57164955 
464999436 735500616 
1 8
462652257 772343790 265750400 891881158 830567300 431223014 151640084 872066310 
2 2
868456133 336921023 
604719879 210210524 
10 9
321299825 353085445 662754002 878680877 745709245 539442130 760060408 993285837 231698703 
882409977 27360464 809048761 375787213 637756015 559969757 544576774 876419740 560793961 
259586022 477847050 754898272 191990108 202504175 491623001 497326655 868221181 713160509 
557970121 75113629 704627009 621587285 226146925 202297888 532850524 105578492 756528285 
30061741 813237956 522551096 222413036 804113512 6662959 351100748 718115119 411456951 
741688976 130429631 302973819 904512319 280001309 567001685 987523622 515950564 376435019 
72296211 737582532 246239948 172287439 256208446 927911211 162189660 425246110 922842971 
671515503 907276600 234693100 532204080 143077350 88832070 271355440 98827456 359394340 
646816938 293001757 959478098 788683262 247240394 117249962 882890858 19365258 963619659 
83712778 208962765 772907804 202843922 236555955 283804032 684757321 517622083 702688602 
