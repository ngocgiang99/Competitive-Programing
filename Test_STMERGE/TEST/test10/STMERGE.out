236560098
237763898
462652257
336921023
83712778
